lib/src/client.dart

```diff
@@ -2199,7 +2221,7 @@ class Client extends MatrixApi {
           requestHistoryOnLimitedTimeline) {
         Logs().v(
             'Limited timeline for ${rooms[roomIndex].id} request history now');
-        runInRoot(rooms[roomIndex].requestHistory);
+        await runInRoot(rooms[roomIndex].requestHistory);
```

Comment:
Why is this getting awaited now? We probably want to do this in the background and not block the sync on a possibly slow history fetch?